src/pages/forside/sections/4-personinfo/3-adresser/visning/AndreAdresser.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import Kilde from "components/kilde/Kilde";
+import { Normaltekst, Undertittel } from "nav-frontend-typografi";
+import { FormattedMessage, useIntl } from "react-intl";
+import { Kontaktadresse as IKontaktadresse } from "../../../../../../types/adresser/kontaktadresse";
+import Kontaktadresse from "./adresser/Kontaktadresse";
+import slettIkon from "../../../../../../assets/img/Slett.svg";
+import Modal from "nav-frontend-modal";
+import { Fareknapp, Flatknapp } from "nav-frontend-knapper";
+import Alert, { AlertType } from "../../../../../../components/alert/Alert";
+import { fetchPersonInfo, slettKontaktadresse } from "../../../../../../clients/apiClient";
+import { PersonInfo } from "../../../../../../types/personInfo";
+import { useStore } from "../../../../../../store/Context";
+
+interface Props {
+  kontaktadresse: IKontaktadresse;
+}
+
+const AndreAdresser = (props: Props) => {
+  const { formatMessage: msg } = useIntl();
+  const { kontaktadresse } = props;
+
+  const [, dispatch] = useStore();
+  const [slettLoading, settSlettLoading] = useState<boolean>();
+  const [slettAlert, settSlettAlert] = useState<AlertType | undefined>();
+  const [visSlettModal, settVisSlettModal] = useState<boolean>(false);
+
+  const apneSlettModal = () => {
+      settVisSlettModal(true);
+  };
+
+  const lukkSlettModal = () => {
+      settVisSlettModal(false);
+  };
+
+  const getUpdatedData = () =>
+    fetchPersonInfo().then((personInfo) => {
+      dispatch({
+        type: "SETT_PERSON_INFO_RESULT",
+        payload: personInfo as PersonInfo,
+      });
+    });
+
+  const onSlettSuccess = () => {
+    lukkSlettModal();
+  };
+
+  const slettPdlKontaktadresse = () => {
+    settSlettLoading(true);
+
+    slettKontaktadresse()
+      .then(getUpdatedData)
+      .then(onSlettSuccess)
+      .catch((error: AlertType) => settSlettAlert(error))
+      .then(() => settSlettLoading(false));
+  };
+
+  return <>
+    <div className="underseksjon__header underseksjon__divider">
+        <Undertittel>
+            <FormattedMessage id={"adresse.overskrift.ovrige"}/>
+        </Undertittel>
+    </div>
+    {
+        kontaktadresse?.kilde === "pdl" && (
+            <>
+                {kontaktadresse && (<Kontaktadresse kontaktadresse={kontaktadresse} tittel={"adresse.kontaktadresse.nav"}/>)}
+                <button onClick={apneSlettModal} className="kilde__lenke lenke">
+                  <span className="kilde__icon">
+                    <img src={slettIkon} alt="Ekstern lenke"/>
```

Comment:
Er det behov for alt-tekst her?

Reply:
Fjernet

Reply:
Viste seg å være påkrevd, har satt den tilbake i egen branch